source/includes/webhooks/_candidate_events.md

```diff
@@ -1202,3 +1202,66 @@ See web hook [common attributes](#common-attributes).
 The New Prospect Application event occurs when a new prospect application is created.
 
 See web hook [common attributes](#common-attributes).
+
+## Candidate anonymized
+
+This web hook will fire when a particular candidate receives an anonymize event. Anonymize events may occur from the [Anonymize Endpoint](#put-anonymize-candidate) in Harvest or may be configured in the GDPR page in the Greenhouse application. An anonymize web hook consists of the action type, the id of the candidate who had properties anonymized, and the information that was anonymized. 
+
+```json
+{
+  "action": "candidate_anonymized",
+  "payload": {
+    "candidate_id": 37031511,
+    "anonymized_fields": ["full_name", "current_title"]
+  }
+}
+```
+
+### Anonymized Fields
+
+A list of possible anonymized fields are provided. If the item in "attribute" is received in the anonymized_fields array, the corresponding changes in notes have been applied in Greenhouse. Note that some of these fields may not be accessible via other web hooks or via Harvest.
+
+| Attribute | Anonymize Process |
+|------------|--------|
+| `activity_items` | Destroy all items in the candidate's activity feed. |
+| `addresses` | Destroy all values from `candidate.addresses` |
+| `all_offer_versions` | For each of this candidate's applications, destroy all offers. (For legacy reasons, this will also raise an [Offer Deleted](#offer-deleted) web hook if one is configured.) |
+| `application_questions` | For each of this candidate's applications, destroy all answers to all application questions. |
```

Comment:
This refers to the job post questions, right? Would it be correct to specify those as `application.answers`?